components/camel-rest-openapi/src/main/java/org/apache/camel/component/rest/openapi/RestOpenApiProcessor.java

```diff
@@ -109,6 +109,14 @@ public boolean process(Exchange exchange, AsyncCallback callback) {
         if (m instanceof RestOpenApiConsumerPath rcp) {
             Operation o = rcp.getConsumer();
 
+            String consumerPath = rcp.getConsumerPath();
+
+            //if uri is not starting with slash then remove the slash in the consumerPath from the openApi spec
+            if(consumerPath.startsWith("/") && !uri.startsWith("/"))
```

Comment:
Add space after if

Reply:
And use same code style as elsewhere, ie curly bracket on same line etc

Reply:
sure thanks, also I see that I forgot to use the variable  consumerPath while calling HttpHelper.evalPlaceholders.